### ISSUE TITLE
Feature request: in-built or dowload-able text file of highlights that is editable and formatable

### ISSUE THREAD

**Issue by ronved** [jimmejardine/qiqqa-open-source#231]

**Comment by GerHobbelt** [comment]
Good idea. The end goal is for all Qiqqa metadata to be exportable and usable by external applications and annotations are part of the metadata managed by Qiqqa.

Regrettably work on PDF rendering, OCR task and Qiqqa Sniffer take precedence (these are main application features and all severely outdated/b0rky ATM) so it may be a while before this feature is available unless someone else chimes in, sorry.

**Comment by ronved** [comment]
Please please somehow help to put the annotations to onenote atleast.
